contracts/ssnlist.scilla

```diff
@@ -457,7 +476,7 @@ procedure CalculateTotalWithdrawal(withdraw: Pair BNum Uint128)
       | False => 
       end
     end
-  | None => (* Won't reach *)
+  | None => throw
```

Comment:
This was unnecessary as `current_deleg` is set to the initiator in `CompleteWithdrawal`. This also does not address the comment in 6.12 which I assume was the intention with this change.
```suggestion
  | None => (* Won't reach *)
```